### PR TITLE
Documentation typo on Spinner component

### PR DESCRIPTION
**Pull request by Latikeshnikam**
Fixes for Documentation typo on Spinner component #5436

### CODE REVIEW

**Comment by Latikeshnikam** [comment]
#5436

**Comment by Latikeshnikam** [comment]
#5457 5436

File: www/src/pages/components/spinners.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import SpinnerButtons from '../../examples/Spinner/Buttons';
 # Spinners
 
 Spinners can be used to show the loading state in your projects.
-Spinners can be used as a
+Spinners can be used as an indicator which notify user that some stuffs are being about to load on the page. It is better to show spinners instead of displaying blank pages before user dropping its interest and navigate.
```
Comment (jquense) [inline www/src/pages/components/spinners.mdx:16]:
```suggestion
```

**Comment by jquense** [comment]
doesn't feel like an attempt to make a useful contribution given the added wording, but not outright spam either. Giving you the benefit of the doubt the correct fix is to remove the extra fragment

**Comment by Latikeshnikam** [comment]
The extra fragments has been removed.

**Comment by Latikeshnikam** [comment]
#5436

**Review by kyletsang — APPROVED** [review]

**Pull request merged**